astropy/utils/tests/test_diff.py

```diff
@@ -54,6 +54,12 @@ def test_diff_types():
                                "    (str) b> '1.0'\n"
                                "           ? +   +\n")
 
+def test_diff_numeric_scalar_types():
+    """ Test comparison of different numeric scalar types. """
+    f = io.StringIO()
+    assert not report_diff_values(1.0, 1, fileobj=f)
+    out = f.getvalue()
+    assert out.lstrip('u') == '  (float) a> 1.0\n    (int) b> 1\n'
```

Comment:
Why does this one test need `.lstrip('u')`? I don't see it used in the other tests.

Reply:
@pllim I just copied _your own_ code from a previous function: https://github.com/astropy/astropy/pull/7444/files#diff-e98489b2b61bd9ec323e65176107ed37R53

Reply:
@pllim Do you know did you put `lstrip()` there?

Reply:
@mcara , mine was a copy-and-paste oversight from `io.fits`. Please remove that one too. Thank you for your *continued* patience! :wink: